### ISSUE TITLE
No prefix support for bar

### ISSUE THREAD

**Issue by XeniaSiskaki** [josdejong/mathjs#1072]
Would it be possible to add the short prefixes for `bar`? We are in need of using `mbar`

**Comment by josdejong** [comment]
Yes sure. Just thinking I guess then we should allow both short and long prefixes on the unit bar? Like `millibar` and `mbar`?

**Comment by XeniaSiskaki** [comment]
Well, it's up to you!

**Comment by josdejong** [comment]
ha ha yeah of course

This is one of the first units that needs both short and long prefixes at the same time, most units have an abbreviation (like `kilometer` and `km`, but no `kilom`), so this is sort of a unique case.

**Comment by josdejong** [comment]
This is fixed now in `v4.1.0`
